### PR TITLE
Supertom acct type

### PR DESCRIPTION
**Pull request by monkeysecurity**
Replaces #536.

Updating the reports dropdown in ui.html for the new url param.

Other minor updates.

@supertom

### CODE REVIEW

**Comment by supertom** [comment]
@monkeysecurity Thanks for cleaning up my tabs and print statements!  I'll be sure to catch those in the future.

File: dart/lib/component/dashboard_component/dashboard_component.dart [hunk 1]
```diff
@@ -30,6 +30,7 @@ class DashboardComponent {
         'regions': '',
         'technologies': '',
         'accounts': '',
+        'accounttypess': '',
```
Comment (cstewart87) [inline dart/lib/component/dashboard_component/dashboard_component.dart]: Is this a typo?

File: dart/lib/component/dashboard_component/dashboard_component.dart [hunk 2]
```diff
@@ -42,6 +43,7 @@ class DashboardComponent {
...
```
Comment (cstewart87) [inline dart/lib/component/dashboard_component/dashboard_component.dart]: Same ? as above, typo?

File: dart/lib/component/dashboard_component/dashboard_component.dart (hunk 1 above)
Comment (supertom) [inline dart/lib/component/dashboard_component/dashboard_component.dart]: @cstewart87 it definitely is, thanks for catching!

File: dart/lib/component/dashboard_component/dashboard_component.dart (hunk 2 above)
Comment (supertom) [inline dart/lib/component/dashboard_component/dashboard_component.dart]: @cstewart87 ditto.

**Comment by supertom** [comment]
@monkeysecurity, @cstewart87 caught two typos of mine (misspelling of accounttypes).  How should we handle?  Another PR?

**Comment by monkeysecurity** [comment]
Sure!

**Pull request merged**